pages/index.js

```diff
@@ -1,7 +1,28 @@
+import MovieCard from "@/components/MovieCard";
+import useSWR from "swr";
+
+const fetcher = async (URL) => {
+  const res = await fetch(URL);
+  const data = await res.json();
+  return data;
+};
+
 export default function HomePage() {
+  const { data: movies, isLoading } = useSWR("/api/movies", fetcher);
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>MovieStar</h1>
+      <ul>
+        {movies?.map((movie) => (
+          <li key={movie.id}>
+            <MovieCard
+              title={movie.title}
+              director={movie.director}
+              image={movie.image}
+            />
+          </li>
```

Comment:
very nice! you decoupled the listing and the MovieCard component. Top quality here ✨